package.json

```diff
@@ -15,8 +15,10 @@
     "git graph"
   ],
   "scripts": {
+    "clean": "pnpm run -r clean ",
     "build": "pnpm build:esbuild && pnpm build:types",
-    "build:esbuild": "pnpm run -r clean && tsx .esbuild/build.ts",
+    "build:esbuild": "pnpm clean && tsx .esbuild/build.ts",
+    "build:vite": "pnpm clean && tsx .vite/build.ts",
```

Comment:
```suggestion
```

Vite build is smaller because it's not building everything esbuild does, and does not have sourcemaps either. 
When compared with the same output (esm), esbuild is smaller than vite (~ half) (and would be much faster, but I just deleted the unrelated files. It's already 10x faster building 4x the things than vite)

<img width="802" alt="image" src="https://github.com/user-attachments/assets/bdb1ff7f-b961-4243-a755-51ba8719e84f" />

```
nr build:vite  69.49s user 7.45s system 172% cpu 44.722 total
nr build:esbuild  6.30s user 1.10s system 85% cpu 8.701 total
```

As we're publishing with esbuild, I don't see a need to keep a vite build in packages, confusing the contributors.